package.json

```diff
@@ -15,12 +14,15 @@
     "start": "node ./docs/server.js",
     "test": "jest",
     "test-watch": "jest --watch=all",
+    "lint": "eslint src docs",
     "build": "npm run build:babel && npm run build:copy-files",
     "build:babel": "babel ./src --ignore *.spec.js --out-dir ./dist",
     "build:copy-files": "cp ./src/styles.css ./dist/; cp ./README.md ./dist/",
-    "build:docs": "webpack --config ./docs/webpack.prod.config",
+    "build:docs": "rm -rf ./docs/build; webpack --config ./docs/webpack.prod.config",
+    "deploy": "bash deploy.sh",
     "prebuild": "rm -rf ./dist",
-    "lint": "eslint src docs"
+    "prepublish": "npm run prebuild; npm run build",
+    "pretest": "npm run lint"
```

Comment:
this is actually awesome. Force ppl to use damn linter. Nobody runs it locally I guess